monitoring/uss_qualifier/scd/simulator/operational_intents_flight_authorisation_data_generator.py

```diff
@@ -0,0 +1,449 @@
+                
+        if len(self.all_test_results) == 1:
+            if "Planned" in self.all_test_results:
+                # The result of both tests is planned
+                if_conflict_with_flight_explanation = KnownIssueFields(test_code = 'nominal_test', relevant_requirements = ['A operational intent that has no time or space conflict should be planned by the USS'], severity= 1, subject='Operational Intent provided should be planned successfully', summary ="The operational intent details provided were generated in such a way that they should have been planned.", details = "The co-ordinates of the 4D Opeational intent does not conflict with any existing operational intent in the area and the processing result should be a successful planning of the intent.")             
+
+                all_known_issue_fields['Rejected']= common_error_notification
+                all_known_issue_fields['Failed']= common_error_notification
+                if self.expected_operational_intent_processing_result != "ConflictWithFlight":
+                    all_known_issue_fields['ConflictWithFlight']= if_conflict_with_flight_explanation
+        else:
+
+            common_conflict_with_flight_explanation = KnownIssueFields(test_code = 'flight_authorisation_test', relevant_requirements = ['A complete and correct flight authorisation data should be provided by the USSP.'], severity= 1, subject='Flight authorisation data is incorrect and operational intent should not be processed', summary ="Invalid flight authorisation data was provided and therefore the operational intent should not have been planned or submitted to the DSS", details = "All flight authorisation data should be validated by the USSP before submitting the Operational Intent to the DSS. In this case, the flight authorisation is not valid and processing should have returned a error.")
+                    
+            if self.expected_flight_authorisation_processing_result =="Rejected":
+                
+                if incorrect_field == 'uas_serial_number':                 
+                    if_planned_explanation = KnownIssueFields(test_code = 'flight_authorisation_test', relevant_requirements = ['A correct UAS Serial number is equired by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 1'], severity= 1, subject='UAS Serial Number provided is incorrect', summary ="The UAS serial number provided in the injection attempt was incorrect", details = "The UAS serial number is not as expressed in the ANSI/CTA-2063 Physical Serial Number format and should be rejected.")                
```

Comment:
Just the relevant requirement: `relevant_requirements = ['ANNEX IV of Commission Implementing Regulation (EU) 2021/664, requirement 1']`